commit e7ec03271a69d340f8b16ce1dbd664dac04de412
Author: MrFlashMode <mr_flashmode@mail.ru>
Date:   Sat Jul 29 07:39:21 2023 +0300

    fully disable r2_aa_kernel, because its shit

diff --git a/src/Layers/xrRender/xrRender_console.cpp b/src/Layers/xrRender/xrRender_console.cpp
index 19cf561..19af4c9 100644
--- a/src/Layers/xrRender/xrRender_console.cpp
+++ b/src/Layers/xrRender/xrRender_console.cpp
@@ -217,7 +217,7 @@ float ps_r2_ls_ssm_kernel = .7f; // r2-only
 float ps_r2_ls_bloom_threshold = 1.f; // r2-only
 Fvector ps_r2_aa_barier = {.8f, .1f, 0}; // r2-only
 Fvector ps_r2_aa_weight = {.25f, .25f, 0}; // r2-only
-float ps_r2_aa_kernel = .5f; // r2-only
+float ps_r2_aa_kernel = .0f; // r2-only
 float ps_r2_mblur = .0f; // .5f
 int ps_r2_GI_depth = 1; // 1..5
 int ps_r2_GI_photons = 16; // 8..64
@@ -1007,7 +1007,7 @@ void xrRender_initconsole()
 	CMD4(CCC_Float, "r2_sun_lumscale_amb", &ps_r2_sun_lumscale_amb, 0.0, +3.0);
 
 	CMD3(CCC_Mask, "r2_aa", &ps_r2_ls_flags, R2FLAG_AA);
-	CMD4(CCC_Float, "r2_aa_kernel", &ps_r2_aa_kernel, 0.3f, 0.7f);
+	CMD4(CCC_Float, "r2_aa_kernel", &ps_r2_aa_kernel, 0.0f, 0.0f);
 	CMD4(CCC_Float, "r2_mblur", &ps_r2_mblur, 0.0f, 1.0f);
 
 	///////lvutner
